openwisp_monitoring/device/templates/admin/config/device/change_form.html

```diff
@@ -334,7 +334,9 @@ <h2>{% trans 'Interface status' %}: {{ interface.name }}</h2>
                   <div class="form-row">
                       <label>{% trans 'Bridge Members' %}:</label>
                       <div class="readonly">
-                          {{ interface.bridge_members|join:", " }}
+                          {% for element in interface.bridge_members %}
+                          <a class="bridge-member" href="javascript:scrollToElement('bridge-member{{ forloop.counter }}')">{{ element }}</a>{% if not forloop.last %}, {% endif %}
```

Comment:
Replace `bridge-member{{ forloop.counter }}` with `status-if-{{ interface.name }}`.

Reply:
it worked well and fine when replacing it with `status-if-{{ element }}`